### ISSUE TITLE
Charm position and right click to dismiss

### ISSUE THREAD

**Issue by thegabriestein** [ffMathy/Shapeshifter#340]
The charms letting you know you've copied something are really good! Small and non-obtrusive, good icon language/messaging, and reminds you that you've copied multiple things from the same source recently.

Small thing, but does it make sense to have it appear in the upper right corner instead of lower left? ShareX and many other notifications appear in lower right. So when I screenshot using ShareX, the thumbnail from a successful screenshot shows up, and it copies it to the clipboard, which causes a charm which pops up right over it haha. Example:

![](https://i.imgur.com/0CmSswa.png)

Lower left isn't that bad though, maybe I'm exaggerating and wouldn't like it as much if it was in upper right where the overlay appears. Thoughts?

Also, **bigger more important thing if it's not too difficult**:
Potentially add a left or right click to dismiss it if it's possible? Then if it's covering something, you don't have to wait for it to go away to access something in the background.

**Comment by thegabriestein** [comment]
Yeah like #224, left side is a no go for basically anything. Right side is the best for sure. In this case it's just whether the top or the bottom corner. Bottom corner is probably fine if the click-to-dismiss feature is added though. That's the primary solution I think. 

Not sure if mouse moving towards it would be good. Maybe to reduce opacity when mouse is near would be good, and click to dismiss? That would make it really gentle interruption wise.

**Comment by ffMathy** [comment]
I think reducing opacity when mouse is near, and then removing it entirely if mouse is over.
